.changeset/ninety-rings-suffer.md

```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-solidity': minor
+---
+
+`GovernorVotesSuperQuorumFraction`: Add a governance extension to enable super quorum with fractional voting.
```

Comment:
`GovernorVotesSuperQuorumFraction` is not about fractional voting, its about the superQuorum being defined as a fraction of the totalSupply of voting token.

```suggestion
`GovernorVotesSuperQuorumFraction`: Add a variant of the `GovernorSuperQuorum` extensions where the super quorum is expressed as a fraction of the total supply.
```